press/press/doctype/marketplace_app/marketplace_app.py

```diff
@@ -547,7 +547,7 @@ def site_installs(self):
 
 	@dashboard_whitelist()
 	def listing_details(self):
-		github_repository_url = frappe.get_value("App Source", {"app": self.app}, "repository_url")
+		github_repository_url = frappe.get_value("App Source", {"app": self.app,"team": self.team}, "repository_url")
```

Comment:
Check if it's public as well. What if the team have a fork for testing?

Reply:
Yep. Thanks.
Did not think about it.